relive-simulator-core/src/commonMain/kotlin/xyz/qwewqa/relive/simulator/core/stage/strategy/complete/CompleteStrategy.kt

```diff
@@ -344,7 +344,7 @@ class CompleteStrategy(val script: CsScriptNode) : Strategy {
     }
 
     private fun drawCard(): CsAct {
-        if (drawPile.isEmpty()) {
+        if (drawPile.count() < 5) {
```

Comment:
Don't really like how this function is being modified. I would rather make this function accept a parameter for the number of cards to be drawn or some variation thereof and change the call sites.